src/components/EventControlledContent.tsx

```diff
@@ -7,6 +8,15 @@ interface EventControlledContentProps {
 
 export const EventControlledContent = ({ waitForEvent, children }: EventControlledContentProps): JSX.Element => {
   const [shouldRender, setShouldRender] = useState(waitForEvent === undefined)
+  const contentDiv = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
```

Comment:
Two things on this:

1. The ref here will always be `null` as it's not actually used anywhere.

2. While we used this approach in the spike, in the "real" iteration let's use a different pattern and not do this in `EventControlledContent` at all. Instead, you can refer to `ContentBlock.tsx` for an alternative pattern we can follow (though open to other suggestions as well)

Reply:
Ok, I see this is what you were mentioning [here](https://github.com/openstax/k12/issues/124#issuecomment-1113603883) about having the math (and tooltip) rendering pushed down to the component leaves.

Reply:
You should be able to remove all changes to this file in your PR without impacting your test cases (e.g. these changes are all unnecessary given your other code)